src/model2D.h

```diff
@@ -182,25 +193,44 @@ void Model<T>::write(string filename, GDALDataType data_type) {
 }
 
 template <typename T> void Model<T>::print() {
-  int nx16 = cols >> 4;
-  int nx32 = cols >> 5;
-  int ny16 = rows >> 4;
-  int ny32 = rows >> 5;
-  cout << "       ";
-  for (int i = 0; i < 16; i++) {
-    cout << " " << setw(8) << nx32 + i * nx16 << " ";
-  }
-  cout << "\n";
+    if ((rows > 16) || (cols > 16)) {
```

Comment:
16 row models are currently used by the turkey nest screening to optimise the 16 x 16 cell search areas. This may need to change if the turkey nest screening algorithm is modified to include search areas of variable sizes.